src/context/mod.rs

```diff
@@ -282,6 +282,15 @@ impl Context {
         raw::notify_keyspace_event(self.ctx, event_type, event, keyname)
     }
 
+    #[cfg(feature = "experimental-api")]
+    pub fn current_command_name(&self) -> &str {
+        unsafe {
+            CStr::from_ptr(raw::RedisModule_GetCurrentCommandName.unwrap()(self.ctx))
```

Comment:
Will crash on redis prior to 6.2 
(https://github.com/redis/redis/pull/8792)
I suggest to check if available, and if not return "N/A" etc., or even better to allow it to fail.